.gitignore

```diff
@@ -5,6 +5,8 @@ CNAME
 .travis.yml
 .idea/
 .vscode/
+tmp/
```

Comment:
What writes to `tmp`, and is there a way to change it so it doesn't? I'd prefer that.

Reply:
As of your last changes, nothing. Previously the link checker deposited a `.log` file in there.